git_theta/hooks/post-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
I think you need a `__init__.py` file for the `__path__` you use later to work in a non-editable install.

Reply:
Just an empty `__init__.py`, right? Nothing can be imported since the files are just shell scripts. 

Aside from that, is this `__path__` method the best way of doing this?

Reply:
It looks like the current recommended way to do this is

1) Update the setup.py to include these files in the `package_data` dict
2) Include these files in a MANIFEST.in
3) iterate through importlib.resources.files to find the files https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files
4) Treat them as files with importlib.resoures.as_file https://docs.python.org/3/library/importlib.resources.html#importlib.resources.as_file

So I think this would end up doing a read+write instead of just a copy.

The above is the most general way to do it. I think you way also works if we set `zip_safe` to false in the setup (but then it is weird that you have this hooks import when there isn't really anything to import)

Reply:
Did some testing and it doesn't seem like adding MANIFEST.in was necessary. Adding the scripts to `package_data` resulted in the hooks being put into the installed package. Also, `importlib.resources.as_file` gives back a path so we can still just copy with `shutil`.

I am fairly inexperienced with packaging in python so I may have missed something here. The way I was testing was just running `pip install .` from the git-theta repo and then verifying that commands that copy the hooks into place were actually doing this.